services/queue/src/workclaimer.js

```diff
@@ -53,17 +53,17 @@ class WorkClaimer extends events.EventEmitter {
   async claim(taskQueueId, workerGroup, workerId, count, aborted) {
     let claims = [];
     let done = false;
-
-    const hintPoller = this.getHintPoller(taskQueueId);
+    let hintPoller;
 
     aborted.then(() => done = true);
     // As soon as we have claims we return so work can get started.
     // We don't try to claim up to the count, that could take time and we risk
     // dropping the claims in case of server crash.
     while (claims.length === 0 && !done) {
+      hintPoller = this.getHintPoller(taskQueueId);
```

Comment:
placing it back where it was before to allow new poller to be created when existing gets destroyed inside the queue..

although I'm not able to reproduce it in the tests :-/